src/app/common/duck/actions.ts

```diff
@@ -1,9 +1,13 @@
-import { createActions } from 'reduxsauce';
+export const ALERT_SUCCESS = 'ALERT_SUCCESS';
+export const ALERT_ERROR = 'ALERT_ERROR';
+export const ALERT_CLEAR = 'ALERT_CLEAR';
 
-const { Creators, Types } = createActions({
-  alertSuccess: ['success'],
-  alertError: ['alertMessage'],
-  alertClear: [],
-});
-
-export { Creators, Types };
+export function alertSuccess(text) {
```

Comment:
Can we enforce with TS that `text` here can only be a string? We've had multiple issues with `errors` getting shoved into these methods and it blowing up at runtime. It'd be great if we could lean on TS more to check this stuff when possible at compile time.